templates/taxi/manufacturer_list.html

```diff
@@ -8,6 +9,11 @@ <h1>
     </a>
   </h1>
 
+  <form action="" method="get" class="form-inline">
+    {{ manufacturer_search_form|crispy }}
+    <input class="btn btn-outline-primary" type="submit" value="🔎">
+  </form>
```

Comment:
The same code in all templates maybe create "include template" and move this code? After include this code here. Just set all forms to the same name